src/core/wrap.ts

```diff
@@ -11,5 +13,5 @@ export type IfSingle<T extends string, Yes, No> = StripEscapes<T> extends `${inf
 
 export const wrap = (s: string | Input<any>) => {
   const v = s.toString()
-  return v.replace(/^\\/, '').length === 1 ? v : `(${v})`
+  return /(?<!\\)(\().*(?<!\\)(\))/.test(v) || v.replace(/^\\/, '').length === 1 ? v : `(${v})`
```

Comment:
Can we simplify this or combine the conditions, do you think?

Reply:
got it, updated, and combined to one regexp.